traffic_ops/traffic_ops_golang/cdn/dnssec.go

```diff
@@ -208,20 +208,14 @@ func DeleteDNSSECKeys(w http.ResponseWriter, r *http.Request) {
 	}
 	defer inf.Close()
 
-	key := inf.Params["name"]
-
-	riakCluster, err := riaksvc.GetRiakClusterTx(inf.Tx.Tx, inf.Config.RiakAuthOptions)
-	if err != nil {
+	cluster, err := riaksvc.GetPooledClusterTx(inf.Tx.Tx, inf.Config.RiakAuthOptions)
+	if nil != err {
```

Comment:
Style nitpick: the `if( 0 ==` pattern is an artifact of C, and IMO generally harder to read. It's more idiomatic in Go and arguably easier to read to put the variable first, `if foo == nil {`.